src/stores/WalletStore.js

```diff
@@ -46,6 +49,8 @@ export default class WalletStore {
   onNakaAccountChange = async (account) => {
     const { loggedIn, network, address, balance } = account;
 
+    this.nbotMethods = window.naka.eth.contract(getContracts().NakaBodhiToken.abi).at(getContracts().NakaBodhiToken.address);
```

Comment:
nit: `nbotContract` probably makes more sense as a name here since you are creating an instance of `contract` at the address.

in web3 1.0, after you create the `Contract` instance, you have to call `contract.methods.myMethod()` which is why I created an `nbotMethods` to reference `contract.methods`. but in web3 < 1.0, the methods are at the same level as the contract functions.

Reply:
create default class variable: `nbotMethods = undefined` so we are aware of what vars can be accessed by the class.